webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -60,6 +60,8 @@
     <servlet-mapping>
         <servlet-name>jersey-servlet</servlet-name>
         <url-pattern>/api/atlas/*</url-pattern>
```

Comment:
Can we remove this? I think admin APIs will break if we remove this.
Can you check where we use admin APIs?

Reply:
this will handled by restriction rules at LB.